src/ui/components/DownloadsManagerView/DownloadItem.tsx

```
@@ -182,51 +182,51 @@ const DownloadItem = ({
           </Box>
           <Box display='flex' fontScale='c1'>
             {expired && (
-              <ActionButton onClick={handleRemove}>
+              <ActionButton onClick={handleRemove} aria-label={t('downloads.item.remove')}>
```

ESLint

[error] Replace `·onClick={handleRemove}·aria-label={t('downloads.item.remove')}` with `⏎················onClick={handleRemove}⏎················aria-label={t('downloads.item.remove')}⏎··············`

(prettier/prettier)